src/components/Language.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+import { withRouter } from "react-router-dom";
+import { BrowserRouter as Link } from "react-router-dom";
+import styled, { css } from "styled-components";
+import { Java } from "@styled-icons/fa-brands/Java";
+import { Python } from "@styled-icons/simple-icons/Python";
+import Quiz from "./Quiz";
+
+//Style
+const Container = styled.div`
+  display: inline-block;
+  border-radius: 3px;
+  padding: 0.5rem 0;
+  margin: 0.5rem 4rem;
+  background: white;
+  color: black;
+  text-align: center;
+  width: 500px;
+  height: 400px;
+`;
+
+const JavaIcon = styled(Java)`
+  margin: 0.5em 1em;
+  color: rgba(33, 182, 168, 1);
+`;
+
+const PythonIcon = styled(Python)`
+  margin: 0.5em 1em;
+  color: rgba(33, 182, 168, 1);
+`;
+
+function Language() {
+  const [langugage, setLanguage] = useState("");
+  console.log(langugage);
```

Comment:
You have a typo here - which is okay if you're consistently using the typo, but it's probably better to just fix it:

```suggestion
  const [language, setLanguage] = useState("");
  console.log(language);
```